frontend/simple/views/Invite.vue

```diff
@@ -80,49 +118,64 @@ export default {
     }
   },
   methods: {
-    add: async function () {
-      if (this.searchUser) {
-        if (this.searchUser === this.$store.state.loggedIn) {
-          this.self = true
-          return
-        } else {
-          this.self = false
-        }
-        try {
-          let contractId = await namespace.lookup(this.searchUser)
-          if (!this.members.find(member => member.name === this.searchUser)) {
-            this.members.push({name: this.searchUser, contractId: contractId})
-          }
-          this.searchUser = null
-          this.error = false
-        } catch (ex) {
-          this.error = true
+    async add () {
+      if (!this.searchUser) return
+
+      if (this.searchUser === this.$store.state.loggedIn) {
+        this.self = true
+        return
+      } else {
+        this.self = false
+      }
+
+      try {
+        const contractId = await namespace.lookup(this.searchUser)
+        if (!this.members.find(member => member.name === this.searchUser)) {
+          this.members.push({ name: this.searchUser, contractId })
         }
+        this.searchUser = null
+        this.error = false
+      } catch (err) {
+        this.error = true
       }
     },
-    remove: function (index) {
+    remove (index) {
       this.members.splice(index, 1)
     },
-    submit: async function () {
-      for (let i = 0; i < this.members.length; i++) {
-        let member = this.members[i]
-        let events = await backend.eventsSince(member.contractId, member.contractId)
-        let [contract, ...actions] = events.map(e => {
+    async submit () {
+      this.members.forEach(async (member) => {
+        const { contractId } = member
+        const events = await backend.eventsSince(contractId, contractId)
+        const [ contract, ...actions ] = events.map(e => {
           return Events[e.entry.type].fromObject(e.entry, e.hash)
         })
-        let state = contract.toVuexState()
+        const state = contract.toVuexState()
         actions.forEach(action => {
-          let type = action.constructor.name
+          const type = action.constructor.name
```

Comment:
Hmm, so this section somehow did not get converted to using the new `latestContractState` in `js/state.js`. Let's convert it. Also, at some the `latestContractState` itself should be modified to received a starting position (and use `contractId` as the starting position by default).